projects/lkft-android/master.jinja2

```diff
@@ -0,0 +1,69 @@
+{% extends "testcases/android/master.jinja2" %}
+
+{% set timeout_common_tests_after_boot = timeout_common_tests_after_boot|default(20) %}
+
+{% set lava_job_name = "lkft-android-" + KERNEL_BRANCH + '-' + BUILD_NUMBER + '-' + job_name %}
+{% set LAVA_JOB_PRIORITY = TEST_LAVA_JOB_PRIORITY %}
+
+{########## LAVA JOB DEFINITION ##############}
+{% block visibility %}
+  group:
+    - {{TEST_LAVA_JOB_GROUP}}
+{% endblock visibility %}
+
+{% block metadata %}
+  android.build: "{{BUILD_NUMBER}}"
+  android.name: "{{JOB_NAME}}"
+  android.url: "{{REFERENCE_BUILD_URL}}"
+  android.version: "{{ANDROID_VERSION}}"
+{% if ANDROID_VENDOR_FINGERPRINT is defined %}
+  android.build.vendor.fingerprint: "{{ANDROID_VENDOR_FINGERPRINT}}"
+{% endif %}
+{% if ANDROID_GSI_FINGERPRINT is defined %}
+  android.build.gsi.url: "{{ANDROID_GSI_URL}}"
+  android.build.gsi.fingerprint: "{{ANDROID_GSI_FINGERPRINT}}"
+{% endif %}
+{% if TUXSUITE_DOWNLOAD_URL is defined %}
+  tuxsuite.download.url: "{{TUXSUITE_DOWNLOAD_URL}}"
+{% endif %}
+  lkft.build.config: "{{LKFT_BUILD_CONFIG}}"
+  git branch: "{{KERNEL_BRANCH}}"
+  git repo: "{{KERNEL_REPO}}"
+  git commit: "{{KERNEL_COMMIT}}"
+  git describe: "{{KERNEL_DESCRIBE}}"
+  build-url: "{{BUILD_URL}}"
+  toolchain: "{{TEST_METADATA_TOOLCHAIN}}"
+  series: lkft
+{% block job_specific_metadata %}
+{% endblock job_specific_metadata %}
+{% endblock metadata %}
```

Comment:
maybe it would make sense to do like the other projects and create a file in projects/<project>/include/metadata.jinja2 ?

Reply:
I thought of it, but as it's only used here, so no meaning to maintain 2 files.
If the same lines are used by another place in the future, then we'd better to create a separate metadata file, and include it for different places.

Reply:
So if I want to do some android testing with this repository, where should I place my project specific file?

Reply:
in the project specific directory. if it could share some of the lines with other projects, say the metadata lines with the lkft-android project, then create the include file under the include directory, and use the include directive for both projects.

Reply:
fair enough.